.gitignore

```diff
@@ -7,3 +7,4 @@ web-ext-artifacts/
 test/dist/*
 ckeditor-build/build/*
 build/
+,idea
```

Comment:
Please do not add this, add this rule to your global git ignore

Reply:
Yeah Sure @vladikoff . Apart from this is it fine ?